src/options/views/tab-settings/vm-editor.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <section>
     <h3 v-text="i18n('labelEditor')"></h3>
-    <p v-html="i18n('descEditorOptions')" />
+    <p v-html="i18n('descEditorOptions') + ' ' + i18n('descEditorTheme')" />
```

Comment:
Can we use two different `<p>`s to support languages that don't like whitespaces better?

Reply:
No, I will modify the existing message then.